include/coap/pdu.h

```diff
@@ -53,11 +58,14 @@
 #define COAP_REQUEST_DELETE    4
 
 /* CoAP option types (be sure to update check_critical when adding options */
+/* See http://www.iana.org/assignments/core-parameters/core-parameters.xhtml#option-numbers */
 
 #define COAP_OPTION_IF_MATCH        1 /* C, opaque, 0-8 B, (none) */
 #define COAP_OPTION_URI_HOST        3 /* C, String, 1-255 B, destination address */
 #define COAP_OPTION_ETAG            4 /* E, opaque, 1-8 B, (none) */
 #define COAP_OPTION_IF_NONE_MATCH   5 /* empty, 0 B, (none) */
+#define COAP_OPTION_OBSERVE         6 /* E, empty/uint, 0 B/0-3 B, (none) */
```

Comment:
Please, keep the previous order (clustered by specification document -- the option numbers are irrelevant for ordering).